src/Mesh/babylon.abstractMesh.ts

```diff
@@ -763,7 +763,7 @@
             var yaw = -Math.atan2(dv.z, dv.x) - Math.PI / 2;
             var len = Math.sqrt(dv.x * dv.x + dv.z * dv.z);
             var pitch = Math.atan2(dv.y, len);
-            Quaternion.RotationYawPitchRollToRef(yaw + yawCor, pitch + pitchCor, rollCor, this.rotationQuaternion);
+            this.rotationQuaternion = Quaternion.RotationYawPitchRollToRef(yaw + yawCor, pitch + pitchCor, rollCor, this.rotationQuaternion || new Quaternion());

```

Comment:
Having said that - I do think that ToRef functions should not return anything. Might be a bit confusing. But it was merged, so it was also reviewed. 
I think the best solution would be to create the rotationQuaternion beforehand.